packages/terra-application/tests/wdio/application-base/application-base-spec.js

```diff
@@ -0,0 +1,73 @@
+Terra.describeViewports('ApplicationBase', ['small', 'large'], () => {
+  describe('breakpoint', () => {
```

Comment:
They're validating that the correct breakpoint string and locale string are being rendered. The screenshots are the same because I'm using the same test example for both tests.

I just wanted a very high-level test to ensure that the context communication is happening. Should something fail in the future, the strings will either render something else (or not render anything at all).